### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by Roland-Ntwali**
## In this PR i completed the following
For each URL you should create:

A route.

An action in the correct controller.

A view file.

View files should include only placeholders written in plain HTML

Make sure that you added code to handle only the 4 required URLs.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```
Comment (ShadyShawkat) [inline spec/requests/users_spec.rb:7]: - [ ] Kindly make sure to implement tests for the `index` and `show` actions for the `user` controller.
- [ ] Please be mindful of the type of tests you'll need to implement for each of the actions (three tests per action), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20response%20status,correct%20placeholder%20text.) requirement.

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      get '/posts/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/posts/show'
+      expect(response).to have_http_status(:success)
+    end
```
Comment (ShadyShawkat) [inline spec/requests/posts_spec.rb:15]: - [ ]Kindly make sure to implement the required tests for each of the actions (three tests per action), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20response%20status,correct%20placeholder%20text.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Roland-Ntwali, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @Roland-Ntwali,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Created controller files with correct actions as required 👍🏽
✅ Routes and views created correctly 👍🏽

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**